src/Brick/Widgets/ProgressBar.hs

```diff
@@ -38,16 +39,41 @@ progressBar :: Maybe String
             -- ^ The progress value. Should be between 0 and 1 inclusive.
             -> Widget n
 progressBar mLabel progress =
+    progressBar' mLabel progress (' ', ' ')
```

Comment:
Please order the `customProgressBar` arguments so that the characters come first, allow this to be written
```haskell
progressBar = customProgressBar ' ' ' '
```